.github/workflows/delete_images.yaml

```diff
@@ -0,0 +1,24 @@
+name: Delete old container images
+
+on:
+  schedule:
+    - cron: '0 0 * * 2' # This job runs every Tuesday
+  workflow_dispatch:
+  push:
+    branches:
+      - main
```

Comment:
Hi @esune Thanks for the feedback. So currently this workflow deletes around 100 images in one run, I have set-up this job to trigger every week so that it catches up in few months and comes to 30 days images asap. 

There were 1300 tagged and 50 untagged images before, it has fallen down to 1031 untagged images and 4 untagged images now. Please let me know if you want me increase the limit restrictions so it deletes all images at once.

Reply:
https://github.com/bcgov/traction/pkgs/container/traction-tenant-ui/versions?page=4

Reply:
I took a peek at the registry, a lot of stuff was pruned - excellent. We will need to remove old leftover `pr-xxx` images that have been closed a long time ago to clean it up, we can create a follow-up task once this is completed.